### ISSUE TITLE
WARNING: Too many errors, dumping unit

### ISSUE THREAD

**Issue by jcoffland** [FoldingAtHome/fah-client-pub#75]
**Reported by @bb30994 on 29 Jun 2010 03:02 UTC**
WUs are dumped due to errors even when no errors have occured.

See #73 for complete documentation.

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 29 Jun 2010 19:35 UTC**
bruce wrote:
Note that in this case, I had no internet connection so the results were NOT sent:

```
13:21:05:Sending unit 00's results
13:21:05:Started thread 1 on PID 3572
13:21:05:Completed 1 unit(s)
```

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 20 Jul 2010 17:57 UTC**
I believe this is fixed in v7.0.8
